src/cryptography/x509/base.py

```diff
@@ -1169,5 +1184,69 @@ def build(self, backend: typing.Any = None) -> RevokedCertificate:
         )
 
 
+class Profile(utils.Enum):
+    RFC5280 = 0
+    WebPKI = 1
```

Comment:
This needs a lot more explanation. What verification behaviors differ between them?

Reply:
`WebPKI` here was meant to mirror the CA/B BRs, which are a strict superset of 5280. `RFC5280` on the other hand was supposed to be "just" 5280, i.e. for Internet PKIs that should be 5280-ish compliant but aren't CA/B compliant (things like Microsoft's PKI, Sigstore, etc.).

I can document that, and ensure that `WebPKI` is always the default (and emphasize that users should never really need anything else, unless they're doing bespoke PKI things). That being said I think non-WebPKI things are also somewhat out of scope for the initially discussed MVP, so I could also remove this configurable knob from the Python API entirely for the time being.